packages/devtools/src/schema/typeDefinitions.ts

```diff
@@ -49,4 +49,13 @@ directive @createModel(
 ) on OBJECT
 
 directive @loadModel(id: StreamID!) on OBJECT
+
+input IndexField {
+  path: [String!]!
+}
+
+directive @createIndex(
```

Comment:
I still think we should define indices as an argument on the `@loadModel` and `@createModel` directives to make it explicit indices are associated to models, for example this should be invalid:

```graphql
type Image @createIndices(...) {
  src: String!
  alt: String
}

type Post @createModel(...) {
  cover: Image!
  ...
}
```

Instead the association would be more explicit with something like:

```graphql
type Image {
  src: String!
  alt: String
}

type Post @createModel(..., indices: [{fields: [{path: ["cover", "src"]}]}]) {
  cover: Image!
  ...
}
```

Reply:
Can revisit this after dogfooding

Reply:
I agree with leaving this as-is for now, but let's definitely not forget to revisit this.  I think experimenting (and possibly even A/B testing) with different syntax for index definitions is something we should really consider a priority during the testing and feedback phase before release - and we should be careful to resist the temptation to push this out to a full release before we've gotten proper usability feedback on the API choices we made.  CC @jpham2023 @michaelsena to put this on your radar